easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023a.yml

```diff
@@ -74,3 +74,4 @@ easyconfigs:
       options:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/20238
         from-pr: 20238
+  - MetalWalls-21.06.1-foss-2023a.eb
```

Comment:
This is targetting the wrong EasyBuild release, you need EasyBuild 4.9.2

Reply:
Ups sorry, just realized i had bot files open and added it to the wrong one.
Fixed in [58fb56f](https://github.com/EESSI/software-layer/pull/642/commits/58fb56f2a8f2fb11d5df2b8332388ace82eda993)